test/packages/parallel/terraform_local/data_stream/local/_dev/deploy/tf/main.tf

```diff
@@ -0,0 +1,5 @@
+resource "local_file" "log" {
+  source          = "./files/example.log"
+  filename        = "/tmp/service_logs/file.log"
```

Comment:
This folder is also accessible by the Elastic Agent, so this file created by Terraform will be used to ingest logs into Elasticsearch.